lua/terrortown/lang/en/defibrillator.lua

```diff
@@ -15,6 +15,7 @@ L["defi_error_no_valid_ply"] = "You can't revive this player since they are no l
 L["defi_error_already_reviving"] = "You can't revive this player since they are already reviving."
 L["defi_error_failed"] = "Revival attempt failed. Please try again."
 L["defi_error_player_alive"] = "You can't revive this player since they are already alive."
+L["defi_error_player_disconnected"] = "You can't revive this player since they have disconnected."
```

Comment:
Can you copy this to the other language files and then comment them out in those?

Reply:
also handled in follow-up